<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSettingSystemSet</class>
 <widget class="QWidget" name="BCSettingSystemSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:rgb(235,235,235);  color: black;
 border-radius: 0px;  border: 0px groove gray;}
 QPushButton:hover{background-color:rgb(205, 205, 205); color: black;}
QPushButton{text-align:left;}

</string>
       </property>
       <property name="text">
        <string>主面版</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>374</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>1000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
